ldm/invoke/generator/diffusers_pipeline.py

```diff
@@ -377,6 +392,7 @@ def generate_latents_from_embeddings(self, latents: torch.Tensor, timesteps,
                                          noise: torch.Tensor,
                                          run_id: str = None,
                                          additional_guidance: List[Callable] = None):
+        self._enable_memory_efficient_attention(latents)
```

Comment:
if we're calling this every generate, it should be called not `_enable` but idk `_adjust_memory_efficient_attention` or something like that.

Reply:
Fixed.